instance/management/commands/kill_zombies.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2020 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Management command to clean up any OpenStack VMs that are known to be dead, but
+evidently are still running about.
+"""
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from instance import openstack_utils
+from instance.models.server import OpenStackServer, Status
+
+
+class Command(BaseCommand):
+    """
+    kill_zombies management command class
+    """
+    help = "Cleans up any OpenStack VMs that on record as being terminated, but are evidently still running."
```

Comment:
@arbrandes "that on record" -- I think you meant "that are on record? :)

Reply:
Whoops!  Yes, of course. :)